aleph-client/src/contract/convertible_value.rs

```diff
@@ -200,6 +200,86 @@ where
     }
 }
 
+impl TryFrom<ConvertibleValue> for Option<String> {
```

Comment:
We should ask around if anyone is using Option<T> where T is not a string and accountid

Reply:
I'll ask, and remove the unnecessary code.